aries-site/src/pages/404.js

```diff
@@ -0,0 +1,65 @@
+import React, { useContext } from 'react';
+import Link from 'next/link';
+import { ThemeContext } from 'styled-components';
+import {
+  Box,
+  Button,
+  Grid,
+  Heading,
+  Image,
+  Paragraph,
+  ResponsiveContext,
+} from 'grommet';
+import { Layout } from '../layouts';
+import { Meta } from '../components';
+
+const title = 'Page Not Found';
+
+const Visual = () => {
+  const theme = useContext(ThemeContext);
+
+  return (
+    <Box>
+      <Image
+        src={theme.dark ? '/404-pile-invert.svg' : '/404-pile.svg'}
+        fit="contain"
+      />
+    </Box>
+  );
+};
+
+const Message = () => (
+  <Box align="start" gap="small">
+    <Heading margin="none">Oops, there’s nothing here.</Heading>
+    <Paragraph margin={{ bottom: 'medium', top: 'none' }} size="large">
+      The page you’re looking for could not be found. Error Type: 404.
+    </Paragraph>
```

Comment:
Just a suggestion... soft opinion.

```suggestion
    <Paragraph margin={{ bottom: 'medium', top: 'none' }} size="large">
      The page you’re looking for could not be found.
    </Paragraph>
    <Paragraph margin={{ bottom: 'medium', top: 'none' }} size="large">
      Error Type: 404.
    </Paragraph>
```

Reply:
Cool, I've broken it into 2 paragraphs.